py/specs/insert_bulk.py

```diff
@@ -0,0 +1,34 @@
+import random, string
+
+
+def random_string(length=100):
+    """
+    Generate random string of uppercase ascii characters and digists
+    of length "length".
+    """
+    return ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(length))
+
+
+def gen_args(x):
+    return (x+1, random_string(), )
+
+
+spec = Spec(
+    setup=Instructions(
+        statements=[
+            "create table t_bulk_insert (id integer, value string) with (number_of_replicas=0)",
+        ]
+    ),
+    teardown=Instructions(
+        statements=[
+            "drop table t_bulk_insert",
+        ]
+    ),
+    queries=[
```

Comment:
Just curious: Why didn't you use ``[[load_data]]`` here? The intention for `load_data` was to be able to benchmark bulk inserts =)